.github/workflows/android.yml

```diff
@@ -77,6 +78,49 @@ jobs:
           if-no-files-found: ignore
           path: ${{ runner.temp }}/artifacts/
 
+  # Running Android emulator directly on the runner and not using Docker
+  run-emulator:
+    needs: build-llm-demo
+    runs-on: amz2023.linux.4xlarge
+    env:
+      ANDROID_NDK_VERSION: r26c
+    steps:
+      - name: Setup SSH (Click me for login details)
```

Comment:
what is clickable?

Reply:
In GH actions page.
Example: https://github.com/pytorch/executorch/actions/runs/10081671587/job/27874241679?pr=4364

I don't think the docker part will apply though:
            All testing is done inside the container, to start an interactive session run:
              docker exec -it $(docker container ps --format '{{.ID}}') bash

Reply:
Ah ok, I copy it paste it, it seems useful to have a shell ready in case we want to login to debug